### PR TITLE
includes-resource-explorer

### PR DESCRIPTION
**Pull request by anaghapamidi**
This update includes:

- Improving the content structure for this article. 
- Added Resource Explorer.

### CODE REVIEW

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc [hunk 1]
```diff
@@ -1,90 +1,186 @@
...
-You can filter the scanned results using Status, Category, Severity, Tags and Code Status.
+image::projects.png[width=800]
+
+==== Browse Scan Results
+
+Each time you access *Code Security > Projects*, you view the latest scan results of a repository. To view a specific repository scan result, you can browse through the list of integrated repositories or search for the repository.
+
+* Browse across repositories and resources.
++
+You can browse multiple integrated repositories and use the icons associated with each repository for an easy find. In this example, you see multiple repositories integrated on Code Security.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc]: you can select only a single repo at a time

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc [hunk 2]
```diff
@@ -1,90 +1,186 @@
...
+
+Prisma Cloud monitors each security violation of a resource within a repository and generates contextualized scanned results of each resource as a resource block.
+//+
+//image::[width=600]
 +
-image::browse-repositories.png[width=600]
-* Scanned resources are grouped by the path of a folder. You can browse across multiple paths within the a repository.
+Each scan result in a resource block gives you extensive information on:
+
+1. Resource Name
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc]: make it bold maybe?

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc [hunk 3]
```diff
@@ -96,31 +192,51 @@ image::fix-scan.png[width=600]
...
+//+
+//image::[width=600]
+
+==== Other Actions on Scan Results
 
-. The suppressed result appears with the Suppress filter.
+On *Code Security > Projects > More Actions*, you can perform additional actions to enable you to view richer scan results of your repositories.
 +
-//image::[width=500]
+image::[width=800]
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan.adoc]: it's shown in the stage env, remove it

**Review by GiladMark — COMMENTED** [review]
@anaghapamidi hey, good job on this! I think you should add an image of 2-3 examples of it. it's now live in prod so it will be easier for you

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**